build/pkgs/ecm/spkg-configure.m4

```diff
@@ -34,4 +36,5 @@ SAGE_SPKG_CONFIGURE([ecm], [
         ], [sage_spkg_install_ecm=yes])
     ])
     m4_popdef([SAGE_ECM_MINVER])
+    AC_SUBST(SAGE_ECMBIN, $SAGE_ECM_BIN)
```

Comment:
```suggestion
    AC_SUBST(ECM, $ECMBIN)
```

Reply:
nope, this causes an ugly name clash sometimes